src/js/entry.js

```diff
@@ -208,3 +209,33 @@ export function rootApp() {
         );
     }
 }
+
+export function noAccess() {
+    const { store } = spinUpStore();
+    window.insights.chrome.auth.getUser().then(({ entitlements }) => {
+        // rhel has different entitlements key and URL partial
+        entitlements.rhel = entitlements.smart_management;
+        const path = location.pathname.split('/');
+        const apps = [];
```

Comment:
The list of apps that require entitlements is better to be added seperately and not part of filter function.

```JS
const apps = Object.keys(entitlements);
```